### PR TITLE
Feature/noproject 0 merge upstream version 2.27.4

### PR DESCRIPTION
**Pull request by javierazabache**

### CODE REVIEW

File: client/src/main/java/com/netflix/conductor/client/http/TaskClient.java [hunk 1]
```diff
@@ -120,11 +119,11 @@ public TaskClient(ClientConfig config, ConductorClientConfiguration clientConfig
      */
     public Task pollTask(String taskType, String workerId, String domain) {
         Preconditions.checkArgument(StringUtils.isNotBlank(taskType), "Task type cannot be blank");
-        Preconditions.checkArgument(StringUtils.isNotBlank(domain), "Domain cannot be blank");
         Preconditions.checkArgument(StringUtils.isNotBlank(workerId), "Worker id cannot be blank");
 
         Object[] params = new Object[]{"workerid", workerId, "domain", domain};
-        Task task = getForEntity("tasks/poll/{taskType}", params, Task.class, taskType);
+        Task task = Optional.ofNullable(getForEntity("tasks/poll/{taskType}", params, Task.class, taskType))
+            .orElse(new Task());
```
Comment (hoaphanFox) [inline client/src/main/java/com/netflix/conductor/client/http/TaskClient.java:126]: huh, I wonder wat they are doing here...

File: mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLMetadataDAO.java [hunk 2]
```diff
@@ -53,19 +54,14 @@ public MySQLMetadataDAO(ObjectMapper om, DataSource dataSource, Configuration co
     }
 
     @Override
-    public String createTaskDef(TaskDef taskDef) {
+    public void createTaskDef(TaskDef taskDef) {
```
Comment (hoaphanFox) [inline mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLMetadataDAO.java:57]: this is weird...

File: README.md [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
 [![License](https://img.shields.io/github/license/Netflix/conductor.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![NetflixOSS Lifecycle](https://img.shields.io/osslifecycle/Netflix/conductor.svg)]()
 
 ## Community
 [![Gitter](https://badges.gitter.im/netflix-conductor/community.svg)](https://gitter.im/netflix-conductor/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) Please feel free to join our Gitter for questions and interacting with the community.
 
+[Access here other Conductor related projects made by the community!](/RELATED.md) - Backup tool, Cron like workflow starter, Docker containers...
+
+## Builds
+Conductor builds are run on Travis CI [here](https://travis-ci.org/Netflix/conductor).
```
Comment (hoaphanFox) [inline README.md:20]: can probaly remove this or put or jenkin.

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-77**

**Comment by pramaha** [comment]
conductor-workers changes to support these changes : https://github.com/fsa-streamotion/bedrock-k8s-conductor-workers/pull/12

**Comment by jason-shiao** [comment]
:star: PR built and available in a preview environment **fsa-streamotion-conductor-pr-77**

**Pull request merged**